.github/workflows/gradle-docker-ci.yml

```diff
@@ -0,0 +1,34 @@
+##
+# Created by brightSPARK Labs
+# www.brightsparklabs.com
+##
+
+name: Gradle Docker CI
+
+on:
+  push:
+    branches: [ feature, develop, master ]
+  pull_request:
+    branches: [ feature, develop, master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 11
+      uses: actions/setup-java@v2
+      with:
+        java-version: '11'
+        distribution: 'temurin'
+    - name: Run Unit Tests
+      run: ./gradlew build
+    - name: Publish
+      if: github.ref == 'refs/heads/master'
+      run: |
+        mkdir -p ~/.gradle
+        echo "gradle.publish.key=${GRADLE_PUBLISH_KEY}" >> ~/.gradle/gradle.properties
+        echo "gradle.publish.secret=${GRADLE_PUBLISH_SECRET}" >> ~/.gradle/gradle.properties
```

Comment:
This way of referencing the GRADLE_PUBLISH_X variables won't work. See how to do it here: https://github.com/brightsparklabs/gradle-docs/pull/7/files